horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/UtilTest.java

```diff
@@ -9,19 +9,58 @@
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.junit.TestProfile;
 import io.quarkus.test.oidc.server.OidcWiremockTestResource;
+import org.jboss.logging.Logger;
+import org.jboss.logmanager.Level;
+import org.jboss.logmanager.LogContext;
+import org.jboss.logmanager.formatters.PatternFormatter;
+import org.jboss.logmanager.handlers.OutputStreamHandler;
 import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+import java.io.UnsupportedEncodingException;
 import java.time.Instant;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.*;
 
 @QuarkusTest
 @QuarkusTestResource(PostgresResource.class)
 @QuarkusTestResource(OidcWiremockTestResource.class)
 @TestProfile(HorreumTestProfile.class)
 public class UtilTest {
 
+    public static class StringHandler extends OutputStreamHandler{
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+        public StringHandler(){
+            setOutputStream(baos);
+            this.setFormatter(new PatternFormatter("%m"));
+            this.setLevel(Level.ALL);
+            this.setAutoFlush(true);
+            LogContext.getLogContext().getLogger("").addHandler(this);
+        }
+
+        public String getLog(){return baos.toString();}
+
+        @Override
+        public void close() throws SecurityException {
+            LogContext.getLogContext().getLogger("").removeHandler(this);
+            super.close();
+            try {
+                baos.close();
+            } catch (IOException e) { /* meh */}
+        }
+    }
 
+    @Test
+    public void toInstant_nulls() throws UnsupportedEncodingException {
+        try(StringHandler handler = new StringHandler()){
+            assertNull(Util.toInstant(null),"null input should return null");
+            assertNull(Util.toInstant(""),"empty string should return null");
+            assertNull(Util.toInstant(" "),"blank string should return null");
+            assertFalse(handler.getLog().contains("DateTimeParseException"),handler.getLog());
```

Comment:
Is this `,handler.getLog()` as assert message wanted or just a leftover?

Reply:
The assert ensures none of those cases logged the exception for parsing becasue they are all valid uses of the function. It also adds an example for checking the logged output which is something we probably need to cover in other test conditions (where logging that something happens is necessary).